shared/teams/team/rows/empty-row.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import * as Styles from '../../../styles'
+import * as Kb from '../../../common-adapters'
+import * as Types from '../../../constants/types/teams'
+import * as ChatTypes from '../../../constants/types/chat2'
+import * as Container from '../../../util/container'
+import * as Constants from '../../../constants/teams'
+import * as TeamsGen from '../../../actions/teams-gen'
+import * as Chat2Gen from '../../../actions/chat2-gen'
+
+type Props = {
+  type: 'channelsEmpty' | 'channelsFew' | 'members' | 'subteams'
+  teamID: Types.TeamID
+  conversationIDKey?: ChatTypes.ConversationIDKey
+  notChannelMember?: boolean
+}
+const icon: {[K in Props['type']]: Kb.IconType} = {
+  channelsEmpty: 'icon-empty-team-small-96',
+  channelsFew: 'icon-empty-channels-103-96',
+  members: 'icon-empty-people-search-112-96',
+  subteams: 'icon-empty-subteams-164-96',
+}
+
+const buttonLabel = {
+  channelsEmpty: 'Create channels',
+  channelsFew: 'Create a channel',
+  members: 'Add/Invite people',
+  subteams: 'Create a subteam',
+}
+
+const useSecondaryAction = (props: Props) => {
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+  const teamID = props.teamID
```

Comment:
`const {teamID} = props`

Reply:
I find those harder to read for no gain when you're only destructuring one element.